etc/fibo-vue/src/views/Courses.vue

```diff
@@ -1,313 +1,111 @@
+                        <p>It will consist of 5 main modules:</p>
+                        <ul>
+                            <li><b>A. Knowledge Graph Business Case and Market Adoption</b> [2:15]
+                                <ul>
+                                    <li><b>A1.</b> Knowledge Graph Perspective On Data</li>
+                                    <li><b>A2.</b> Knowledge Graph Industry Use Cases</li>
+                                    <li><b>A3.</b> Knowledge Graphs Takeaways</li>
+                                </ul>
+                            </li>
+                            <li><b>B. Knowledge Graph Technical Framework</b> [3:00]
+                                <ul>
+                                    <li><b>B1.</b> Knowledge Graph Enterprise Architecture Overview</li>
+                                    <li><b>B2.</b> <b>B3.</b> & <b>B6.</b> Knowledge Graph Data Languages (RDF, RDFS, SKOS)</li>
+                                    <li><b>B4.</b> Knowledge Graph Modeling Language (OWL)</li>
+                                    <li><b>B5.</b> Knowledge Graph Query language (SPARQL)</li>
+                                    <li><b>B8.</b> Knowledge Graph Enterprise Architecture Ecosystem</li>
+                                </ul>
+                            </li>
+                            <li><b>C. FIBO Industry Model review</b> [2:00]
+                                <ul>
+                                    <li><b>C1.</b> Structure</li>
+                                    <li><b>C2.</b> Content</li>
+                                    <li><b>C3.</b> Infrastructure</li>
+                                </ul>
+                            </li>
+                            <li><b>D. Knowledge Graph Hands-on Exercises</b> [2:00]
+                                <ul>
+                                    <li><b>D1.</b> Swaps in Finance</li>
+                                    <li><b>D2.</b> Toxic byproducts in Agriculture</li>
+                                    <li><b>D3.</b> Automotive manufacturing</li>
+                                </ul>
+                            </li>
+                            <li><b>E. Knowledge Graph Course Key Takeaways</b> [0:15]
+                                <ul>
+                                    <li><b>E1.</b> Survey</li>
+                                    <li><b>E2.</b> Accreditation</li>
+                                </ul>
+                            </li>
+                        </ul>
+
+
+                        <h3><span>Course Palette</span></h3>
+                        <p><em>Open Knowledge graph and FIBO Fundamentals Course</em> is just part of our educational offer. Our full offer of courses covers different areas of Knowledge Graph Industry. These are shown below as a mind map that interlinks different learning topics.</p>

```

Comment:
Still not sure why upper case "O" and "K" with lower case "g" in "Open Knowledge graph"

Reply:
Agreed - I think this one just fell through the cracks.